src/HousingCoordinates.cpp

```
@@ -0,0 +1,154 @@
+            {dai::HousingCoordinateSystem::VESA_B, {50.0f, 0.0f, 0.0f}},
+            {dai::HousingCoordinateSystem::VESA_C, {5.0f, 39.299999999999997f, 32.590000000000003f}},
+            {dai::HousingCoordinateSystem::VESA_D, {25.0f, 39.299999999999997f, 32.590000000000003f}},
+            {dai::HousingCoordinateSystem::VESA_E, {45.0f, 39.299999999999997f, 32.590000000000003f}},
+        }
+    },
+    {
+        "OAK-4-S-FF",
+        {
+            {dai::HousingCoordinateSystem::CAM_A, {25.0f, 10.199999999999999f, 67.620000000000005f}},
+            {dai::HousingCoordinateSystem::FRONT_CAM_A, {25.0f, 10.199999999999999f, 69.5f}},
+            {dai::HousingCoordinateSystem::IMU, {12.699999999999999f, 30.550000000000001f, 59.93f}},
```

Cppcheck (2.20.0)

[style] The function 'getHousingCoordinates' is never used.

(unusedFunction)